CHANGELOG.md

```diff
@@ -6,8 +6,12 @@
 ### Feature:
 * Introduce new SequentialHostInit view allocation property [\#7229](https://github.com/kokkos/kokkos/pull/7229)
 
+### CUDA:
+* Experimental support for unified memory mode (intended for Grace-Hopper etc.) [\#6823](https://github.com/kokkos/kokkos/pull/6823)
+
 ### Bug Fix
```

Comment:
It is clashing with the way we usually organize our change log

Reply:
But its the same as we did for MI300 stuff ;-)

Reply:
It had a `Backend and Architecture Enhancements:` section header